developer-docs/content-watcher/README.md

```diff
@@ -1,356 +1,186 @@
-
-### Tech Stack <a name="tech-stack"></a>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://nestjs.com/">NestJS</a></li>
-    <li><a href="https://nodejs.org/">Node.js</a></li>
-    <li><a href="https://www.typescriptlang.org/">TypeScript</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Data store</summary>
-  <ul>
-    <li><a href="https://redis.io/">Redis (ioredis)</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Frameworks and Libraries</summary>
-  <ul>
-    <li><a href="https://docs.nestjs.com/techniques/queues">BullMQ (NestJS BullMQ Integration)</a></li>
-    <li><a href="https://github.com/hapijs/joi">Joi</a></li>
-    <li><a href="https://axios-http.com/">Axios</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Polkadot and DSNP Integration</summary>
-  <ul>
-    <li><a href="https://polkadot.js.org/">Polkadot API (@polkadot/api)</a></li>
-    <li><a href="https://github.com/LibertyDSNP/spec">DSNP (Activity Content, Frequency Schemas, ParquetJS)</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Testing</summary>
-  <ul>
-    <li><a href="https://jestjs.io/">Jest</a>
-        <ul>
-          <li><a href="https://kulshekhar.github.io/ts-jest/">ts-jest</a></li>
-          <li><a href="https://github.com/visionmedia/supertest">Supertest</a></li>
-          <li><a href="https://github.com/stipsan/ioredis-mock">ioredis-mock</a></li>
-        </ul>
-      </li>
-  </ul>
-</details>
-
-<details>
-  <summary>Formatting</summary>
-  <ul>
-    <li><a href="https://prettier.io/">Prettier</a></li>
-    <li><a href="https://typescript-eslint.io/">TypeScript ESLint</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Build and Deployment</summary>
-  <ul>
-    <li><a href="https://github.com/motdotla/dotenv">Dotenv</a></li>
-    <li><a href="https://www.docker.com/">Docker</a></li>
-    <li><a href="https://docs.docker.com/compose/">Docker Compose</a></li>
-    <li><a href="https://docs.nestjs.com/cli/overview">Nest CLI</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
-
-### Key Features
-
-#### API
-
-- **Parse DSNP Messages on Frequency**
-- **Send content to registered webhooks**
-- **Simple Schema and MSA Id based filtering**
-
-#### Scanner API
-
-- **Start, Stop, and restart scanning**
+## 📖 About the Project <a name="about-project"></a>
 
-<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+The Content Watcher Service is part of the Gateway suite that provides a Web2-friendly interface for monitoring and retrieving blockchain content. It implements a webhook system that pushes updates to registered endpoints, making it easy to stay synchronized with chain events.
 
-<!-- LIVE Docs -->
+## 🔍 Architecture Overview <a name="architecture-overview"></a>
 
-## 🚀 Live Docs
+The Content Watcher Service provides a webhook-based system for receiving notifications about new content. Even with a crawl request, results are pushed to the webhook rather than retrieved directly.
```

Comment:
The second sentence is a little confusing to me. Thinking either we drop it or explain why we cannot retrieve it directly.